setup.php

```diff
@@ -1574,14 +1570,14 @@ function cloneClicked() {
     <script>
         //jquery-ui tooltip
         $(function() {
-            $('.icon-tooltip').prop( "title", "Click to see more information").tooltip({
+            $('.icon-tooltip').attr({"title": <?php echo xlj('Click to see more information'); ?>, "data-toggle":"tooltip", "data-placement":"bottom"}).tooltip({
                 show: {
                     delay: 700,
                     duration: 0
                 }
             });
-            $('.enter-details-tooltip').prop( "title", "Additional help to fill out this form is available by hovering over labels of each box and clicking on the dark blue help ? icon that is revealed. On mobile devices tap once on the label to reveal the help icon and tap on the icon to show the help section").tooltip();
-            $('.2fa-section-tooltip').prop( "title", "Two factor authentication prevents unauthorized access to openEMR thus improves security. It is optional. More information is available in the help file under Step 2 Database and OpenEMR Initial User Setup Details.").tooltip();
+            $('.enter-details-tooltip').attr({"title": <?php echo xlj('Additional help to fill out this form is available by hovering over labels of each box and clicking on the dark blue help ? icon that is revealed. On mobile devices tap once on the label to reveal the help icon and tap on the icon to show the help section'); ?>, "data-toggle":"tooltip", "data-placement":"bottom"}).tooltip();
+            $('.2fa-section-tooltip').attr({"title": <?php echo xlj('Two factor authentication prevents unauthorized access to openEMR thus improves security. It is optional. More information is available in the help file under Step 2 Database and OpenEMR Initial User Setup Details.'); ?>, "data-toggle":"tooltip", "data-placement":"bottom"}).tooltip();
```

Comment:
Note that the translation functions are not available in this script since there is no database yet. Would revert the xlj change here.